internal/core/data/v2/application/event.go

```diff
@@ -135,6 +135,20 @@ func EventCountByDevice(deviceName string, dic *di.Container) (uint32, errors.Ed
 	return count, nil
 }
 
+// The DeleteAllEvents function will be invoked by controller functions
+// and then invokes DeleteAllEvents function in the infrastructure layer to remove
+// all events
+func DeleteAllEvents(dic *di.Container) errors.EdgeX {
```

Comment:
Please rename all these functions to `DeletePushedEvents` be more explicate on what is being deleted.

Reply:
Renamed.